modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionServiceImpl.java

```diff
@@ -46,6 +46,23 @@
 public class ObjectDefinitionServiceImpl
 	extends ObjectDefinitionServiceBaseImpl {
 
+	@Override
+	public ObjectDefinition addCustomObjectDefinition(
+			Boolean enableComments, Map<Locale, String> labelMap, String name,
```

Comment:
@TarcianaBarros we should use the primitive type `boolean`, instead of `Boolean`